src/olympia/migrations/865-add-contributions-usd.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `stats_contributions` ADD COLUMN `amount_usd` DECIMAL(9, 2);
+UPDATE `stats_contributions` SET `amount_usd` = `amount`;
```

Comment:
The stats pages display contributions in USD. Using just the amount would make them worthless (especially given something like the Yen, where $1 = ¥110). The pages could be changed to display amounts for each currency, but this is a lot simpler.